_includes/head.liquid

```diff
@@ -46,6 +46,25 @@
 
 <!-- Styles -->
 
+<!-- Add scripts/style for Lightbox2 -->
+{% if site.lightbox.enabled %}
+  <link
+    defer
+    rel="stylesheet"
+    type="text/css"
+    href="{{ '/assets/css/lightbox.min.css' | relative_url | bust_file_cache }}"
+  >
+  <script src="{{ '/assets/js/lightbox-plus-jquery.min.js' | relative_url | bust_file_cache }}"></script>
```

Comment:
Maybe this line and the next script tag should be included in [_includes/scripts/imageLayouts.liquid](https://github.com/alshedivat/al-folio/blob/main/_includes/scripts/imageLayouts.liquid) instead.